app/src/main/java/com/example/myanimereport/fragments/HomeFragment.java

```diff
@@ -64,20 +83,57 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         binding.tvCreate.setOnClickListener(this::createOnClick);
         binding.btnMenu.setOnClickListener(this::openNavDrawer);
 
-        // Add entries to the recycler view
-        queryEntries(0);
+        // Search view
+        EditText searchEditText = binding.searchView.findViewById(androidx.appcompat.R.id.search_src_text);
```

Comment:
nit: `import androidx.appcompat.R;`, and use as `binding.searchView.findViewById(R.id.search_src_text);`

Reply:
If I import that it will conflict with `import com.example.myanimereport.R` :/
![image](https://user-images.githubusercontent.com/59420335/127057155-cfe93cc3-3c17-4cd8-a0ce-20510b4fdb23.png)

Reply:
Never mind it works if I just `import com.example.myanimereport.R` and use `binding.searchView.findViewById(R.id.search_src_text)`